release 0.6.0 (#451) (#452)
redhat-developer/intellij-kubernetes: README.md

 - Support Creation and Deletion of new Kubernetes resources on Cluster
 - Navigate into Kubernetes workloads, services, pods and nodes
 - Switching contexts and namespaces within Kubernetes

>  - Follow logs from your clusters

Reviewer: ```suggestion
 - Follow logs from the connected cluster resource
```

#### Delete Resources
You may delete any resource that is listed in the resource tree by choosing "Delete" in the context menu.

#### Follow logs

> Intellij-Kubernetes allows you to follow the logs of Pods, Jobs and Builds. 

Reviewer 2: extension is for VSCode, plugin is to be used here

> A dedicated tab continuously displays the messages that the chosen resource sends to its log.  

Reviewer: ```suggestion
There is a dedicated tab to display the log messages for the selected Kubernetes resource.
```
